sparse_merkle_tree.js

```diff
@@ -19,6 +19,7 @@ const {
     in_memory_smt_verify,
     in_memory_smt_calculate_root,
 } = require("./bin-package/index.node");
+const { isInclusion } = require('./utils');
```

Comment:
I would make the function name more clear, something like `isInclusionProofForQueryKey` or something similar

Reply:
Done.